### DISCUSSION TITLE
Won't search newegg-ca

### DISCUSSION THREAD

**Discussion by SuperFlyBry** [jef/streetmerchant#1252] [Help]
## Steps to Reproduce

1.
2.
3.
4.

## Environment

**OS:**
**.env file:**

```dotenv

```

**Comment by SuperFlyBry** [comment]
Sorry it was status code error 403

**Comment by jef** [comment]
Probably banned from searching frequently.

**Comment by SuperFlyBry** [comment]
How do I fix this? I seen on a twitch stream some guy searches newegg-ca constantly with no issiues

**Comment by jef** [comment]
> How do I fix this? I seen on a twitch stream some guy searches newegg-ca constantly with no issiues

They probably have a lot of proxies.
